docs/detections/alerts-ui-manage.asciidoc

```diff
@@ -87,6 +87,38 @@ To change alert statuses, either:
 * In the Alerts table, select all the alerts you want to change, and then select
 *Take action* -> *Close selected*, *Open selected*, or *Mark in progress*.
 
+[float]
+[[signals-to-cases]]
+=== Add alerts to cases
+From the Alerts table, you can attach one or more alerts to a case by using the *Add to case* button. From here, you can choose to <<signals-to-new-cases, add the alert to a new case>> or <<signals-to-existing-cases, attach it to an existing one>>. There is no limit to the number of alerts that you can add and you can choose alerts from any rule type - custom or pre-built. If the case has been configured to sync its status with associated alerts, the alert's status will update after it's been added. If this setting isn't enabled, the alert's status will not change after you've added it.
+
+NOTE: After you have added an alert to a case, you can only remove it through the the Elastic Security Cases API.
+
+[role="screenshot"]
+image::images/add-alert-to-case.gif[width=80%][height=80%][Shows how to add an alert to a case]
+
+[float]
+[[signals-to-new-cases]]
+==== Add an alert to a new case
+Follow these steps to add an alert to a new case:
+
+. Select **Add to case -> Add to a new case**.
+. In the flyout, give your case a name, add relevant tags, and include a case description.
+. Specify whether you want to sync the statuses of associated alerts. This setting is enabled by default to enforce asynchronous communication between alerts and cases and can be toggled on or off at any time.
```

Comment:
> This setting is enabled by default to enforce asynchronous communication between alerts and cases and can be toggled on or off at any time.

nit: _asynchronous communication_ makes me think of bidirectional communication between cases and alerts (maybe that's just an assumption though). When the setting is enabled Cases will control the status of the alerts. Alerts don't really communicate back to Cases.

Maybe something like:

`This setting is enabled by default to allow Cases to control the state of attached alerts' status. Whenever the Case status changes the status of the attached alerts will be updated to match it. It can be toggled on or off at any time.`

Reply:
I agree with @jonathan-buttner. It also makes me think the same.